src/util/system.cpp

```diff
@@ -935,6 +935,14 @@ static bool GetConfigOptions(std::istream& stream, const std::string& filepath,
     return true;
 }
 
+bool IsConfSupported(KeyInfo& key, std::string& error) {
```

Comment:
5e744f423838fe7d45453541271bc1a07cd62eac nit
```suggestion
bool IsConfSupported(const KeyInfo& key, std::string& error) {
```